ddl/index.go

```diff
@@ -1675,6 +1677,8 @@ func writeOneKVToLocal(writer ingest.Writer,
 	index table.Index, sCtx *stmtctx.StatementContext, writeBufs *variable.WriteStmtBufs,
 	idxDt, rsData []types.Datum, handle kv.Handle) error {
 	iter := index.GenIndexKVIter(sCtx, idxDt, handle, rsData)
+	unlock := writer.LockForWrite()
```

Comment:
Can we move this lock/unlock operation out to writeChunkToLocal and cover all for loop?